api/src/main/java/io/strimzi/api/kafka/model/listener/arraylistener/GenericKafkaListenerConfiguration.java

```diff
@@ -229,6 +230,17 @@ public void setIpFamilies(List<IpFamily> ipFamilies) {
         this.ipFamilies = ipFamilies;
     }
 
+    @Description("Whether to create the bootstrap service or not." +
+            "This field can be used with the `loadBalancer` type listener.")
```

Comment:
Since we support this only with load balancers, you should also add some validation to https://github.com/strimzi/strimzi-kafka-operator/blob/main/cluster-operator/src/main/java/io/strimzi/operator/cluster/model/ListenersValidator.java